### PR TITLE
Fix`internal/log/format` to match the test template

### PR DESCRIPTION
**Pull request by hlts2**
Signed-off-by: hlts2 <hiroto.funakoshi.hiroto@gmail.com>

### Description:

I fixed `internal/log/foramt` to match the test template.

### Environment:

- Golang Version: 1.14
- Docker Version: 19.03.5
- Kubernetes Version: 1.17.3
- NGT Version: 1.9.1

### Types of changes:

- [ ] Bug fix [type/bug]
- [ ] New feature [type/feature]
- [x] Add tests [type/test]
- [ ] Security related changes [type/security]
- [ ] Add documents [type/documentation]
- [ ] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [x] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [x] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [x] I have added tests and benchmarks to cover my changes.
- [x] I have ensured all new and existing tests passed.
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: internal/log/format/format_test.go [hunk 1]
```diff
@@ -23,153 +23,160 @@ import (
...
+			},
 		},
 
 		{
-			name: "returns LTSV when str is LTSv",
+			name: "returns LTSV when str is `LTSv`",
 			str:  "LTSv",
-			want: LTSV,
+			want: want{
+				LTSV,
```
Comment (kevindiu) [inline internal/log/format/format_test.go:164]:
```suggestion
				want: LTSV,
```

**Comment by kevindiu** [comment]
LGTM

File: internal/log/format/format_test.go (hunk 1 above)
Comment (hlts2) [reply internal/log/format/format_test.go:164]: thank you for your review.
I will fix it soon.

**Comment by vankichi** [comment]
@kpango @rinx 
Could you please review, if you have time?

**Pull request merged**